docker/central-ledger/default.json

```diff
@@ -1,6 +1,23 @@
 {
   "PORT": 3001,
   "HOSTNAME": "http://central-ledger",
+  "DATABASE": {
+    "DIALECT": "mysql",
+    "HOST": "central_ledger",
+    "PORT": 3306,
+    "USER": "central_ledger",
+    "PASSWORD": "password",
+    "SCHEMA": "central_ledger",
+    "POOL_MIN_SIZE": 10,
+    "POOL_MAX_SIZE": 10,
+    "ACQUIRE_TIMEOUT_MILLIS": 30000,
+    "CREATE_TIMEOUT_MILLIS": 30000,
+    "DESTROY_TIMEOUT_MILLIS": 5000,
+    "IDLE_TIMEOUT_MILLIS": 30000,
+    "REAP_INTERVAL_MILLIS": 1000,
+    "CREATE_RETRY_INTERVAL_MILLIS": 200,
+    "DEBUG": false
+  },
   "RUN_MIGRATIONS": true,
   "AMOUNT": {
```

Comment:
I know that we're overriding it in helm values / config, but can you please update these to 18,4 here as well?

Reply:
ok